### PR TITLE
Implement wearable general dialog

### PR DESCRIPTION
**Pull request by singa2000**
[Issue] N/A
[Problme] Wearable genenral dialog is wrong.
[Solution] Implement the general dialog

Signed-off-by: cookie <cookie@samsung.com>

### CODE REVIEW

File: src/css/profile/wearable/changeable/theme-circle/popup.less [hunk 1]
```diff
@@ -71,11 +72,24 @@
 			padding: 8 * @unit_base 18 * @unit_base 0 23 * @unit_base 0;
 			color: @color_button_bottom_text;
 			margin: 8 * @unit_base 0;
-			height: 100 * @unit_base;
-			width: 60 * @unit_base;
-			background-color: @bg_color_button_bottom;
+			height: 121 * @unit_base;
+			width: 73 * @unit_base;
+			background-color: @color_popup_side_button_bg;
+			mask-image: url('./images/Popup/tw_ic_popup_btn_bg.png');
```
Comment (HunseopJeong) [inline src/css/profile/wearable/changeable/theme-circle/popup.less]: Use double quotes.

File: src/css/profile/wearable/changeable/theme-circle/popup.less [hunk 2]
```diff
@@ -71,11 +72,24 @@
...
 			border: 0;
-			border-radius: 60 * @unit_base;
+
+			&.ui-btn-check {
+				&::before {
+					background-color: @color_popup_button_icon_bg;
+					mask-image: url('./images/Popup/tw_ic_popup_btn_check.png');
```
Comment (HunseopJeong) [inline src/css/profile/wearable/changeable/theme-circle/popup.less]: ditto.

File: src/css/profile/wearable/changeable/theme-circle/popup.less [hunk 3]
```diff
@@ -71,11 +72,24 @@
...
+				}
+			}
+			&.ui-btn-delete {
+				&::before {
+					background-color: @color_popup_button_icon_bg;
+					mask-image: url('./images/Popup/tw_ic_popup_btn_delete.png');
```
Comment (HunseopJeong) [inline src/css/profile/wearable/changeable/theme-circle/popup.less]: ditto.

File: src/css/profile/wearable/changeable/theme-circle/popup.less [hunk 4]
```diff
@@ -71,11 +72,24 @@
...
-			width: 60 * @unit_base;
-			background-color: @bg_color_button_bottom;
+			height: 121 * @unit_base;
+			width: 73 * @unit_base;
+			background-color: @color_popup_side_button_bg;
+			mask-image: url('./images/Popup/tw_ic_popup_btn_bg.png');
 			border: 0;
-			border-radius: 60 * @unit_base;
+
+			&.ui-btn-check {
```
Comment (HunseopJeong) [inline src/css/profile/wearable/changeable/theme-circle/popup.less:81]: 'ui-btn-check' and 'ui-btn-delete' seems to be used commonly.
How about moving the styles to the button.less?

Comment (singa2000) [reply src/css/profile/wearable/changeable/theme-circle/popup.less:81]: In the guide the button just used in the popup.
So the icon name is also "tw_ic_popup_btn_xx".
So the icon size also different with another default button icon.
If you don't mind, I want to keep it in the popup button.

Comment (HunseopJeong) [reply src/css/profile/wearable/changeable/theme-circle/popup.less:81]: Your answer is reasonable!
It would be good to keep it.

**Review by HunseopJeong — APPROVED** [review]
LGTM

File: examples/wearable/UIComponents/contents/assistviews/popup.html [hunk 5]
```diff
@@ -85,18 +85,17 @@ <h2 class="ui-title">
 			<div class="ui-popup-content">
 				Agree?
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/assistviews/popup.html:86]: In debugger it shows font-size 32 px for this statement
Guide says about 30px
Please check @font_size_popup_text

Comment (singa2000) [reply examples/wearable/UIComponents/contents/assistviews/popup.html:86]: Ok. I changed font-size to 30px.
Thanks.

**Pull request merged**